### PR TITLE
Python 3 Cleanup

### PR DESCRIPTION
**Pull request by tisto**

### CODE REVIEW

**Comment by tisto** [comment]
@jenkins-plone-org please run jobs

File: src/plone/restapi/testing.py [hunk 1]
```diff
@@ -57,20 +56,6 @@
 else:
     PLONE_5 = True  # pragma: no cover
```
Comment (thet) [inline src/plone/restapi/testing.py:57]: I think we can also drop this, right?

Comment (thet) [reply src/plone/restapi/testing.py:57]: I think the ``PLONE_VERSION`` should also not be necessary, some lines above.

**Review by thet — APPROVED** [review]
LGTM.

I love this massive reduction of code.
IMO, this is a necessary step to move forward efficiently without the burden of keeping everything backwards compatible. I hope the FTW people also see it like that and are OK with this change.

**Comment by thet** [comment]
@tisto one minor thing: please add a news/towncryer entry.

File: src/plone/restapi/testing.py (hunk 1 above)
Comment (tisto) [reply src/plone/restapi/testing.py]: @thet Good catch! Removed it.

**Comment by tisto** [comment]
@jenkins-plone-org please run jobs

**Review by mauritsvanrees — APPROVED** [review]
LGTM. Great to see this code size reduction!

More could be done, but that can also happen in another PR. At least I still see lots of uses of `six`, and especially `six.PY2` which could be removed.

**Comment by tisto** [comment]
@mauritsvanrees @thet thanks for the review! Yeah, I am wondering what's the best approach here. Lots of small PRs that are easy to review or one big bang PR that removes everything. I guess the current PR is big enough already, so I wouldn't add more. Next to do would be indeed to remove all the six imports and stuff.

**Comment by tisto** [comment]
@thet tbh I don't think an official FWT review is necessary here. We decided that Plone 6 will be Python 3 only and this is just a cleanup of a core package that is used only in Plone 6. Though, of course, I am happy for any review and approval (official or not) from the @plone/framework-team or its members.

**Comment by mauritsvanrees** [comment]
Agreed, `plone.restapi` master is for Plone 6 only since a few days or weeks, so dropping code for earlier versions is fine.

And fine to merge as is, and not create an even larger PR.

**Comment by ale-rt** [comment]
I am also fine with this cleanup and welcome similar actions in the future for other packages.

P.s.: And yes, I would also like other @plone/framework-team members to express their ideas and maybe organize a meeting...

File: plone-4.3.x.cfg [hunk 2]
```diff
@@ -1,39 +0,0 @@
-[buildout]
```
Comment (wesleybl) [inline plone-4.3.x.cfg:1]: I see that `make update` is used to update the versions of this package. The `Makefile` file still tries to download cfgs from unsupported versions. It is necessary to remove these files from the `Makefile`:

https://github.com/plone/plone.restapi/blob/at-cleanup/Makefile#L27-L28

**Review by wesleybl — CHANGES_REQUESTED** [review]

File: plone-4.3.x.cfg (hunk 2 above)
Comment (tisto) [reply plone-4.3.x.cfg:1]: @wesleybl thanks for the review! I am aware of this problem and I plan to put some effort into fixing it. Though, not in this PR.

**Comment by tisto** [comment]
@ale-rt I am open for a fwt call. Do you see this as a blocker for a merge here? I'd prefer to merge this sooner than later because it touches quite a few files and will lead to merge conflicts when this becomes a long-running PR.

File: plone-4.3.x.cfg (hunk 2 above)
Comment (wesleybl) [reply plone-4.3.x.cfg:1]: OK @tisto Just for the record, I created #1144

**Review by wesleybl — COMMENTED** [review]
Okay. Makefile will be handled in another PR

**Review by wesleybl — APPROVED** [review]

**Comment by ale-rt** [comment]
> @ale-rt I am open for a fwt call. 

That's a great news, I will ask again in the @plone/framework-team mailing list.

> Do you see this as a blocker for a merge here? 

Absolutely not!

> I'd prefer to merge this sooner than later because it touches quite a few files and will lead to merge conflicts when this becomes a long-running PR.

I would approve the PR but the CI complains.
I will indeed approve it when green :)

**Comment by tisto** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**